cli/cmd/alpha.go

```diff
@@ -4,6 +4,8 @@ import (
 	"github.com/spf13/cobra"
 )
 
+// newCmdAlpha creates a new cobra command for the `alpha` command which are
+// used by experimental subcommands for Linkerd
```

Comment:
perhaps we should say that this "contains experimental subcommands" rather than is "used by experimental subcommands"